package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "codejam_keyboard",
+  "version": "1.0.0",
+  "description": "",
+  "main": ".eslintrc.js",
+  "dependencies": {
+    "eslint": "^6.1.0",
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-plugin-import": "^2.18.2"
```

Comment:
The dependencies, required for development (eslint, for example) should be stored in the devDependencies field. 
You find more information on this topic here
https://medium.com/@stalonadsl948/dependencies-vs-devdependencies-926e096a3dee